### ISSUE TITLE
Make slash continuations resolve to a single space

### ISSUE THREAD

**Issue by casey** [casey/just#628]
Recipe slash continuations preserve leading whitespace on the line after the slash. When commands with line continuations are echoed, they contain this extra leading whitespace, which makes them harder to read.

For example, this recipe line:

```
help2man \
  --name 'save and run commands' \
  --manual 'Just Manual' \
  --no-info \
  target/debug/just \
  > man/just.1
```

Gets echoed as:

```
help2man        --name 'save and run commands'  --manual 'Just Manual'  --no-info       target/debug/just     > man/just.1
```

It would be nice to echo whitespace, followed by a line continuation, followed by leading whitespace on the next line as a single space.

This is unlikely to break any recipes, but I think this should be considered a breaking change. It won't change commands, but if line continuations are used within strings, then this would change the amount of whitespace inside those strings.

**Comment by rjsberry** [comment]
It would be great if we could also partially preserve indentation.

Given the recipe:

```make
say-hello:
  docker run alpine \
    sh -c ' \
      speak=hello; \
      echo $speak \
    '
```

I would expect this to be echoed as:

```
$ just say-hello
docker run alpine \
 sh -c ' \
  speak=hello; \
  echo $speak \
 '
hello
```

Rather than:

```
$ just say-hello
docker run alpine \
 sh -c ' \
 speak=hello; \
 echo $speak \
 '
hello
```

**Comment by casey** [comment]
Thanks for the feedback! I think I personally would prefer it to be echoed as 

```
docker run alpine sh -c ' speak=hello; echo $speak'
```

Just to avoid taking up too much space in the terminal. I think it's definitely arguable either way. I'm curious what other people think.

Is there a difference between the last two example outputs?

**Comment by rjsberry** [comment]
My mistake - I had interpretted the original issue as preserving the newline in the recipe. If that is not the case, trimming the the whitespace on the single line of output is certainly an improvement.

The difference in the example outputs is simply the indentation inside the command string run by `sh`.

**Comment by casey** [comment]
> My mistake - I had interpretted the original issue as preserving the newline in the recipe. If that is not the case, trimming the the whitespace on the single line of output is certainly an improvement.

Ahh, gotcha.

> The difference in the example outputs is simply the indentation inside the command string run by `sh`.

Ooop, I see it now!

**Comment by casey** [comment]
Fixed in #635. This is breaking change, so next release will bump the version to `0.6.0`.
